packages/@aws-cdk/aws-glue/lib/connection.ts

```diff
@@ -0,0 +1,130 @@
+import { Construct, IResource, Resource, Aws } from '@aws-cdk/core';
+import ec2 = require('@aws-cdk/aws-ec2');
+import { CfnConnection } from './glue.generated';
+
+export enum ConnectionInputTypes {
+  /**
+   * The type of the connection. Currently, only JDBC is supported; SFTP is not supported.
+   */
+  JDBC = 'JDBC',
+}
+
+
+export interface IPhysicalRequirements {
+  /**
+   * @attribute
+   */
+  readonly availabilityZone?: string;
+
+  /**
+   * @attribute
+   */
+  readonly securityGroupIds?: ec2.ISecurityGroup[];
+
+  /**
+   * @attribute
+   */
+  readonly subnet?: ec2.ISubnet;
```

Comment:
I believe what you want here is a standard couple of properties we use in these cases:

```ts
readonly vpc: ec2.Vpc;

readonly vpcSubnets?: ec2.SubnetSelection;
```

This should be instead of the `subnet` property.